docs/source/index.rst

```diff
@@ -86,10 +86,12 @@ In the :code:`schema` above:
 
 -   :code:`Animal` represents a non-abstract vertex. For relational databases, we think of
     tables as the non-abstract vertices.
--   :code:`name` is a **property field** and :code:`out_Animal_LivesIn` is a **vertex field**,
-    which represent an outbound edge to a vertex in the graph. For graph databases,
-    edges can be reflected from the database schema. However, for relational databases edges have
-    to be manually specified. See :doc:`SQL <databases/sql>` for more information.
+-   :code:`name` is a **property field** which represents a property of the :code:`Animal` vertex.
+    Think of **property fields** as the conceptual equivalent to table columns.
+-   :code:`out_Animal_LivesIn` is a **vertex field** which represents an outbound edge to a vertex
+    in the graph. For graph databases, edges can be reflected from the database schema. However,
```

Comment:
`reflected` is not good word choice here. Consider `generated automatically` or something to that effect. Also consider adding a `currently` in `for relational databases edges have to be manually specified`:
```
for relational databases, edges currently have to be manually specified and cannot be automatically generated from the schema
```